ynotradio/site
# PHP Version Upgrade

> The ynot website is currently powered by PHP verision 5.6 which is end of life and is no longer a supported version.  This requires that the software build of PHP be upgraded on all the hosting servers to maintain security of the systems and assure stability.  The new version of PHP will be 7.x

This branch makes a handful of broad changes:
- To make maintenance easier, all site files are moved to a new /src directory
- A readme was added with instructions for using the Docker containers that were added in #3 
- Changes to PHP scripts to prepare the site for PHP 7 environments
- Updating Docker environment to mirror a PHP 7.4 environment

To verify changes:

- Spin up local dev environment as described in README, and perform smoke test of site at http://localhost:8100/
- From project root directory, run php compatibility check as described in README: `docker run --rm --volume $(pwd):/app vfac/php7compatibility 7.4 ./src -d memory_limit=1G --extensions=php` - should return no errors or warnings
